core/src/main/java/com/coderanch/blackjack/Card.java

```diff
@@ -76,13 +100,15 @@ Suit suit() {
         return this.suit;
     }
 
+    int points() {
+        return this.rank.points;
+    }
```

Comment:
Seeing as this method isn't covered by code coverage, I'm going to assume that it's not used. Please remove it.